extensions/wikia/PortableInfobox/services/PortableInfoboxMobileRenderService.class.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use Wikia\PortableInfobox\Helpers\PortableInfoboxRenderServiceHelper;
+
+class PortableInfoboxMobileRenderService extends PortableInfoboxRenderService {
+	const MEDIA_CONTEXT_INFOBOX_HERO_IMAGE = 'infobox-hero-image';
+	const MEDIA_CONTEXT_INFOBOX = 'infobox';
+
+	protected $imagesWidth = PortableInfoboxRenderServiceHelper::MOBILE_THUMBNAIL_WIDTH;
+
+	/**
+	 * renders infobox
+	 *
+	 * @param array $infoboxdata
+	 *
+	 * @param $theme
+	 * @param $layout
+	 * @param $accentColor
+	 * @param $accentColorText
+	 * @return string - infobox HTML
+	 */
+	public function renderInfobox( array $infoboxdata, $theme, $layout, $accentColor, $accentColorText ) {
```

Comment:
5 arguments is on the edge of being too many. How do you feel about using `array $infoboxdata, array $options`?
https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#Functions_and_parameters